### PR TITLE
fix Slack notifications

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/tell-slack-failed.yml [hunk 1]
```diff
@@ -8,10 +8,13 @@ steps:
   - template: bash-lib.yml
     parameters:
       var_name: bash_lib
+    condition: and(failed(),
```
Comment (cocreature) [inline ci/tell-slack-failed.yml:11]: can we just set this to `always` so we don’t have to keep the two in sync? The check for sending it below should be sufficient

**Review by cocreature — APPROVED** [review]
:facepalm:, sorry for messing this up

File: ci/tell-slack-failed.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/tell-slack-failed.yml:11]: I suppose we can spare the extra second.

**Comment by cocreature** [comment]
This currently fails with "unexpected value condition". We can probably just move the condition inside the template.

**Pull request merged**